phpdotnet/phd/Package/PHP/XHTML.php

```diff
@@ -487,6 +487,10 @@ public function format_type_text($type, $tagname) {
         case "double":
             $href = "language.types.float";
             break;
+        // old name for callable
+        case "callback":
+            $href = "language.types.callable";
+            break;
```

Comment:
This had deliberately been removed. It seems the reintroduction is accidential.

Reply:
Indeed will drop this